src/pages/projects/esg/Index.vue

```diff
@@ -144,6 +157,9 @@ export default {
                 return url;
             }
         },
+        slugify(string) {
```

Comment:
We have `@sindresorhus/slugify` as a depenency already for other path manipulations if you just want to reuse that here.